src/test/components/Navbar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './Navbar.css';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+
+function NavBar() {
+  return (
+    <Router>
+      <nav className=" d-flex justify-content-end align-content-center gap-2 container-fluid py-3 pe-5">
+        <div className="container-fluid ">
+          <h1>Math Magician!</h1>
+        </div>
+        <Link className="text-decoration-none border-end border-2 border-dark px-2  my-auto" to="/">Home</Link>
+        <Link className="text-decoration-none border-end border-2 border-dark px-2  my-auto " to="/calculator">Calculator</Link>
+        <Link className="text-decoration-none   my-auto " to="/qoute">Quote</Link>
+      </nav>
+    </Router>
+  );
+}
+export default NavBar;
```

Comment:
- [ ] All components should be tested. Kindly add unit tests for this Navbar component.

- [ ] Kindly add a test for the Calculator component.

- [ ] There is no need to make a copy of all component and logic files. I can see you have `components.test` and `logic.test` folders containing the same files and code created earlier. We can test the components directly instead of duplicating files. Kindly remove the duplicate files.